test/integration/deploy_trigger_test.go

```diff
@@ -114,37 +106,53 @@ func TestTriggers_manual(t *testing.T) {
 }
 
 func TestTriggers_imageChange(t *testing.T) {
-	testutil.DeleteAllEtcdKeys()
-	openshift := NewTestOpenshift(t)
-	defer openshift.Close()
+	_, clusterAdminKubeConfig, err := testutil.StartTestMaster()
+	if err != nil {
+		t.Fatalf("error starting master: %v", err)
+	}
+	openshiftClient, err := testutil.GetClusterAdminClient(clusterAdminKubeConfig)
+	if err != nil {
+		t.Fatalf("error getting OpenShift cluster admin client: %v", err)
+	}
+	kubeClient, err := testutil.GetClusterAdminKubeClient(clusterAdminKubeConfig)
+	if err != nil {
+		t.Fatalf("error getting OpenShift cluster admin client: %v", err)
+	}
+	openshiftClientConfig, err := testutil.GetClusterAdminClientConfig(clusterAdminKubeConfig)
+	if err != nil {
+		t.Fatalf("error getting cluster admin client config: %v", err)
+	}
+	_, err = testutil.CreateNewProject(openshiftClient, *openshiftClientConfig, testutil.Namespace(), "admin")
+	if err != nil {
+		t.Fatalf("error creating project: %v", err)
+	}
 
-	imageRepo := &imageapi.ImageRepository{ObjectMeta: kapi.ObjectMeta{Name: "test-image-repo"}}
+	imageStream := &imageapi.ImageStream{ObjectMeta: kapi.ObjectMeta{Name: "test-image-stream"}}
 
 	config := deploytest.OkDeploymentConfig(0)
 	config.Namespace = testutil.Namespace()
-	var err error
 
-	watch, err := openshift.KubeClient.ReplicationControllers(testutil.Namespace()).Watch(labels.Everything(), fields.Everything(), "0")
+	watch, err := kubeClient.ReplicationControllers(testutil.Namespace()).Watch(labels.Everything(), fields.Everything(), "0")
```

Comment:
Is there a reason we can't run this test as a project admin?  That would help us make sure that we don't accidentally start restricting our project admins too far.